apps/coscrad-frontend-e2e/src/e2e/resources/photographs/queries/photograph.index.query-flow.e2e.cy.ts

```diff
@@ -0,0 +1,298 @@
+            cy.contains(dummyPhotographTitle);
+        });
+
+        it(`should have a link to the detail view for this photograph`, () => {
+            cy.getByDataAttribute(`${photographId}`).click();
+
+            cy.contains(dummyPhotographTitle);
+
+            cy.location('pathname').should('contain', `/Resources/Photographs/${photographId}`);
+        });
+
+        describe(`the search filter`, () => {
+            const mediaItemForFilterTestCompositeIdentifier =
+                buildDummyAggregateCompositeIdentifier(AggregateType.mediaItem, 17);
+
+            const photographForFilterTestCompositeIdentifier =
+                buildDummyAggregateCompositeIdentifier(AggregateType.photograph, 23);
+
+            const photoTitleWithQDash = 'Q-Photo Haida';
+
+            const haidaPhotoTitleToFind = 'Q-';
+
+            const searchScopes = [`allProperties`, `name`];
+
+            before(() => {
+                cy.seedDataWithCommand(`CREATE_MEDIA_ITEM`, {
+                    aggregateCompositeIdentifier: mediaItemForFilterTestCompositeIdentifier,
+                    title: `Sg_iidllg_uu`,
+                    mimeType: 'image/jpeg',
+                    // Override default value to "remove" property from fsa for image media item
+                    lengthMilliseconds: null,
+                });
+
+                cy.seedDataWithCommand(`PUBLISH_RESOURCE`, {
+                    aggregateCompositeIdentifier: mediaItemForFilterTestCompositeIdentifier,
+                });
+
+                cy.seedDataWithCommand(`CREATE_PHOTOGRAPH`, {
+                    aggregateCompositeIdentifier: photographForFilterTestCompositeIdentifier,
+                    title: photoTitleWithQDash,
+                    languageCodeForTitle: LanguageCode.Haida,
+                    mediaItemId: mediaItemForFilterTestCompositeIdentifier.id,
+                    heightPx: 800,
+                    widthPx: 200,
+                });
+
+                cy.seedDataWithCommand(`PUBLISH_RESOURCE`, {
+                    aggregateCompositeIdentifier: photographForFilterTestCompositeIdentifier,
+                });
+
+                const NUMBER_OF_ENGLISH_PHOTOS = 10;
+
+                const NUMBER_OF_HAIDA_PHOTOS = 15;
+
+                const MEDIA_ITEM_OFFSET = NUMBER_OF_ENGLISH_PHOTOS + NUMBER_OF_HAIDA_PHOTOS;
+
+                seedManyPhotographsInLanguageWithMediaItems(
+                    NUMBER_OF_ENGLISH_PHOTOS,
+                    ID_OFFSET,
+                    MEDIA_ITEM_OFFSET,
+                    LanguageCode.English
+                );
+
+                seedManyPhotographsInLanguageWithMediaItems(
+                    NUMBER_OF_HAIDA_PHOTOS,
+                    ID_OFFSET + NUMBER_OF_ENGLISH_PHOTOS,
+                    MEDIA_ITEM_OFFSET + NUMBER_OF_ENGLISH_PHOTOS,
+                    LanguageCode.Haida
+                );
+            });
+
+            searchScopes.forEach((searchScope) => {
```

Comment:
I realize I introduced this test-builder pattern. I prefer not to do things this way because I don't like to interfere with .only \ .skip. But let's keep with it for the current test. Very good to see the filtering test coverage.